src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,21 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
+  name: "exfmt"
+  link: "https://github.com/lpil/exfmt"
+  isPreInstalled: false
```

Comment:
Same comment as https://github.com/Glavin001/atom-beautify/pull/1726#discussion_r122724679

I would expect there to be a `mix` executable.
Then instead of `args: ['--version'],` it could be `args: ['exfmt', '--version'],`, that is if `mix exfmt --version` works?